OSDOCS-16941: DITA abstracts — OLM catalog/config topics (#115366)
openshift/openshift-docs: operators/understanding/olm-rh-catalogs.adoc

toc::[]

[role="_abstract"]

> {product-title} includes Red Hat-provided Operator catalogs by default for use with Operator Lifecycle Manager (OLM). These catalogs deliver Red Hat, certified, and community Operators through index images in the software catalog.

Reviewer: ```suggestion
Red Hat provides several Operator catalogs that are included with {product-title} by default.
```
